src/modules/fancyzones/lib/Resources.resx

```diff
@@ -216,4 +216,16 @@
   <data name="Span_Across_Zones_Warning" xml:space="preserve">
     <value>The 'Allow zones to span across monitors' option requires all monitors to have the same DPI scaling to work properly.</value>
   </data>
+  <data name="FancyZones_Data_Error" xml:space="preserve">
+    <value>Cannot find path to FancyZones persisted data. Please report the bug to</value>
```

Comment:
Cannot find path to FancyZones persisted data -> Cannot find FancyZones persisted data pah

Reply:
Actually, it would be better: FancyZones persisted data pah not found.